seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2048,9 +2048,8 @@
       "title": "Manipulate Arrays With pop()",
       "description": [
         "Another way to change the data in an array is with the <code>.pop()</code> function.",
-        "<code>.pop()</code> is used to \"pop\" a value off of the end of an array. We can store this \"popped off\" value by assigning it to a variable.",
+        "<code>.pop()</code> is used to \"pop\" a value off of the end of an array. We can store this \"popped off\" value by assigning it to a variable. In other words, <code>.pop()</code> removes the last element from an array and returns that element.",
         "Any type of entry can be \"popped\" off of an array - numbers, strings, even nested arrays.",
-        "For example, for the code<br><code>var oneDown = [1, 4, 6].pop();</code><br>the variable <code>oneDown</code> now holds the value <code>6</code> and the array becomes <code>[1, 4]</code>.",
```

Comment:
According to [@erictleung's comment](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/11092#issuecomment-252470111), should there not be a code snippet here?